spec/rails_helper.rb

```diff
@@ -0,0 +1,82 @@
+require 'webmock/rspec'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.